Mathlib/CategoryTheory/Enriched/Ordinary.lean

```diff
@@ -0,0 +1,138 @@
+/-
+Copyright (c) 2024 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.CategoryTheory.Enriched.Basic
+import Mathlib.CategoryTheory.Monoidal.Types.Coyoneda
+
+/-!
+# Enriched ordinary categories
+
+If `V` is a monoidal category, a `V`-enriched category `C` does not need
+to be a category. However, when we have both `Category C` and `EnrichedCategory V C`,
+we may require that the type of morphisms `X ⟶ Y` in `C` identify to
+`𝟙_ V ⟶ EnrichedCategory.Hom X Y`. This data shall be packaged in the
+typeclass `EnrichedOrdinaryCategory V C`.
+
+In particular, if `C` is a `V`-enriched category, it is shown that
+the "underlying" category `ForgetEnrichment V C` is equipped with a
+`EnrichedOrdinaryCategory V C` instance.
+
+Simplicial categories are implemented in `AlgebraicTopology.SimplicialCategory.Basic`
+using an abbreviation for `EnrichedOrdinaryCategory SSet C`.
+
+-/
+
+universe v' v u u'
+
+open CategoryTheory Category MonoidalCategory
+
+namespace CategoryTheory
+
+variable (V : Type u') [Category.{v'} V] [MonoidalCategory V]
+  (C : Type u) [Category.{v} C]
+
+/-- An enriched ordinary category is a category `C` that is also enriched
+over a category `V` in such a way that morphisms `X ⟶ Y` in `C` identify
+to morphisms `𝟙_ V ⟶ (X ⟶[V] Y)` in `V`. -/
+class EnrichedOrdinaryCategory extends EnrichedCategory V C where
+  /-- morphisms `X ⟶ Y` in the category identify morphisms
+    `𝟙_ V ⟶ (X ⟶[V] Y)` in `V` -/
+  homEquiv {X Y : C} : (X ⟶ Y) ≃ (𝟙_ V ⟶ (X ⟶[V] Y))
+  homEquiv_id (X : C) : homEquiv (𝟙 X) = eId V X := by aesop_cat
+  homEquiv_comp {X Y Z : C} (f : X ⟶ Y) (g : Y ⟶ Z) :
+    homEquiv (f ≫ g) = (λ_ _).inv ≫ (homEquiv f ⊗ homEquiv g) ≫
+      eComp V X Y Z := by aesop_cat
+
+variable [EnrichedOrdinaryCategory V C] {C}
+
```

Comment:
Shouldn't the rest of the file be in the `EnrichedOrdinaryCategory` namespace?

Reply:
The API for enriched categories is in the `CategoryTheory` namespace (and not `CategoryTheory.Enriched`). As this API complements the one on enriched categories, I feel it should be in the same namespace (as long as we keep the prefix `e`).